src/app/(app)/(home)/search-filters/index.tsx

```
@@ -0,0 +1,15 @@
+import { Categories } from './categories';
+import { SearchInput } from './search-input';
+
+interface Props {
+  data: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)